src/components/Form/formActions.ts

```diff
@@ -263,6 +292,38 @@ export const formActions = {
 			})
 		}
 	},
+	isSubmittable () {
+		return function (dispatch, getState): boolean {

```

Comment:
#

Reply:
I've checked in a change that ensures the form's initial state contains `busyFields` as an empty object - it was previously `loading` but it was decided that `busyFields` is more appropriate.